pandas/tseries/tests/test_frequencies.py

```diff
@@ -379,6 +380,17 @@ def test_freq_to_reso(self):
             result = Reso.get_freq(Reso.get_str(Reso.get_reso_from_freq(freq)))
             self.assertEqual(freq, result)
 
+    def test_resolution_bumping(self):
+        Reso = frequencies.Resolution
+
+        self.assertEqual(Reso.bump_to_int(1.5, 'T'), (90, 'S'))
+        self.assertEqual(Reso.bump_to_int(62.4, 'T'), (3744, 'S'))
```

Comment:
can u exercise some more fail cases
and systematically cover cases that should succeed